x-pack/plugins/observability/public/pages/rule_details/index.tsx

```diff
@@ -174,229 +79,107 @@ export function RuleDetailsPage() {
-                        {i18n.translate('xpack.observability.ruleDetails.actionsButtonLabel', {
-                          defaultMessage: 'Actions',
-                        })}
-                      </EuiButton>
-                    }
-                  >
-                    <EuiFlexGroup direction="column" alignItems="flexStart" gutterSize="s">
-                      <EuiButtonEmpty
-                        data-test-subj="editRuleButton"
-                        size="s"
-                        iconType="pencil"
-                        onClick={handleEditRule}
-                      >
-                        <EuiText size="s">
-                          {i18n.translate('xpack.observability.ruleDetails.editRule', {
-                            defaultMessage: 'Edit rule',
-                          })}
-                        </EuiText>
-                      </EuiButtonEmpty>
-                      <EuiButtonEmpty
-                        size="s"
-                        iconType="trash"
-                        color="danger"
-                        onClick={handleRemoveRule}
-                        data-test-subj="deleteRuleButton"
-                      >
-                        <EuiText size="s">
-                          {i18n.translate('xpack.observability.ruleDetails.deleteRule', {
-                            defaultMessage: 'Delete rule',
-                          })}
-                        </EuiText>
-                      </EuiButtonEmpty>
-                    </EuiFlexGroup>
-                  </EuiPopover>
-                </EuiFlexItem>
-              </EuiFlexGroup>,
-            ]
-          : [],
-      }}
-    >
-      <EuiFlexGroup wrap={true} gutterSize="m">
-        <EuiFlexItem style={{ minWidth: 350 }}>
-          {getRuleStatusPanel({
-            rule,
-            isEditable: hasEditButton,
-            requestRefresh: reloadRule,
-            healthColor: getHealthColor(rule.executionStatus.status),
-            statusMessage,
-          })}
-        </EuiFlexItem>
-        <EuiSpacer size="m" />
-        <EuiFlexItem style={{ minWidth: 350 }}>
-          {getRuleAlertsSummary({
-            rule,
-            filteredRuleTypes,
-          })}
-        </EuiFlexItem>
-        <EuiSpacer size="m" />
-        {getRuleDefinition({ rule, onEditRule: () => reloadRule() } as RuleDefinitionProps)}
-      </EuiFlexGroup>
+            <RuleDefinition
+              actionTypeRegistry={actionTypeRegistry}
+              rule={rule}
+              ruleTypeRegistry={ruleTypeRegistry}
+              onEditRule={reloadRule}
+            />
+          </EuiFlexGroup>
 
-      <EuiSpacer size="l" />
-      <EuiTabbedContent
-        data-test-subj="ruleDetailsTabbedContent"
-        tabs={tabs}
-        selectedTab={tabs.find((tab) => tab.id === tabId)}
-        onTabClick={(tab) => {
-          onTabIdChange(tab.id as TabId);
-        }}
-      />
-      {editFlyoutVisible &&
-        getEditAlertFlyout({
```

Comment:
Component methods exported by TriggerActionsUI are JSX components, so why not use JSX syntax